app/src/org/commcare/connect/ConnectManager.java

```diff
@@ -109,7 +114,7 @@ public interface ConnectActivityCompleteListener {
     }
 
     private static ConnectManager manager = null;
-    private ConnectIdStatus connectStatus = ConnectIdStatus.NotIntroduced;
+    public static ConnectIdStatus connectStatus = ConnectIdStatus.NotIntroduced;
```

Comment:
This should stay as a member of the class. If we need it accessible from elsewhere, let's create:
public static ConnectIdStatus getStatus() {
   return getInstance().status;
}